src/components/Modal/index.js

```diff
@@ -87,8 +87,14 @@ class Message extends React.Component {
 
 class Description extends React.Component {
   render() {
+    const { children } = this.props;
+
     // format if the child is just plain text
-    if (typeof this.props.children === 'string') {
+    if (
+      typeof children === 'string' ||
+      (typeof children === 'object' &&
+        !some(false, map(child => typeof child === 'string')))
```

Comment:
Confused by this part. How does `some(false, ...` work? The first argument is a collection, right? Maybe a left over from debugging?

Reply:
@codeluggage we use the functional version of lodash, so we switch the position of the arguments!